CIP8/README.md

```diff
@@ -1,5 +1,5 @@
 ---
-CIP: 7
+CIP: 8
```

Comment:
You would need to change the folder name on this one also

Reply:
I did - it's now under CIP8/ (while Shawn's lengthy CIP folder name was changed to CIP7/)